DebugBot/DebugBotMain.cpp

```diff
@@ -1,17 +1,24 @@
-#include <iostream>
 #include "sc2api/sc2_api.h"
 #include "sc2api/sc2_args.h"
 #include "sc2lib/sc2_lib.h"
 #include "sc2utils/sc2_manage_process.h"
 #include "sc2utils/sc2_arg_parser.h"
 
+#include <Windows.h>
+#include <iostream>
+#include <experimental/filesystem>
+
 #include "LadderInterface.h"
 
 class DebugBot : public sc2::Agent {
 public:
 	virtual void OnGameStart() final {
 		const sc2::ObservationInterface* observation = Observation();
 		std::cout << "I am player number " << observation->GetPlayerID() << std::endl;
+		std::cout << "Executable: " << GetExecutableFullFileName() << std::endl;
+		// if the following statement gives anyone trouble, it might be
+		// that current_path() is a C++17 function.
+		std::cout << "Working directory: " << std::experimental::filesystem::current_path() << std::endl;
```

Comment:
But you've request only c++14 support in #21. Why?

Reply:
I'll try to go through #21 again over the weekend, I think I had a few issues with it, but I'll try to work through that

Reply:
@alkurbatov Previously we weren't using any c++17 features that I'm aware of, so I went with what s2client-api had in their cmake.
With that said, I think I will remove current_path() as per @Cryptyc's request, which means we can maintain c++14 if we want.

Reply:
The real problem with it is that you request a user to have quite fresh compiler (i.e. with c++17 support) which is not the case of various Linux server systems, some LInux desktops and probably some older versions of Windows. As a result the set of OS which this code supports become significantly smaller.
So I agree on it with @Cryptyc